projects/ng-draw-flow/src/lib/components/connections/draft-connection/draft-connection.service.ts

```diff
@@ -38,8 +38,16 @@ export class DraftConnectionService implements OnDestroy {
     protected readonly destroy$ = new Subject<void>();
     private sourceConnector!: DfDataConnector;
 
-    public readonly source$ = new BehaviorSubject<DfPoint>(INITIAL_COORDINATES);
-    public readonly target$ = new BehaviorSubject<DfPoint>(INITIAL_COORDINATES);
+    public readonly source$ = new BehaviorSubject<DfConnectorData>({
+        point: INITIAL_COORDINATES,
+        position: DfConnectorPosition.Right,
+    });
+
+    public readonly target$ = new BehaviorSubject<DfConnectorData>({
+        point: INITIAL_COORDINATES,
+        position: DfConnectorPosition.Left,
+    });
```

Comment:
nit: don't you want to try `signal` with initial value + `computed()`, instead of BehaviorSubject ?